doc/MqttMeter_DE.md

```diff
@@ -6,7 +6,9 @@ Für einen MQTT-Zähler muss dessen `Topic` angegeben werden sowie der `Paramete
 
 Wenn die MQTT-Nachticht im **JSON-Format** geliefert wird, sollte das als `Format` eingestellt werden. Damit lässt sich durch die Angabe des `Pfad für Extraktion` der Zahlenwert sehr leicht [aus der MQTT-Nachricht extrahieren](ValueExtraction_DE.md). Wird der optionale `Pfad für die Zeit-Extraktion` angegeben, wird die aus der MQTT-Nachricht extrahierte Zeit anstelle der aktuellen Zeit für die weitere Verarbeitung verwendet.
 
-Alternativ (oder auch nachgelagert zur JSON-Interpretation) kann ein [regulärer Ausdruck zum Extrahieren](ValueExtraction_DE.md) angegeben werden, falls der Zahlenwert aus einem Text (XML, ...) extrahiert werden muss. Dies gilt auch, wenn die MQTT-Nachricht scheinbar nur die Zahl enthält, diese aber auch einen Zeilenumbruch (CR/LF) beinhaltet.
+Alternativ (oder auch nachgelagert zur JSON-Interpretation) kann ein [regulärer Ausdruck zum Extrahieren](ValueExtraction_DE.md) angegeben werden, falls der Zahlenwert aus einem Text (XML, ...) extrahiert werden muss. Dies gilt auch, wenn die MQTT-Nachricht scheinbar nur die Zahl enthält, diese aber auch einen Zeilenumbruch (CR/LF) beinhaltet oder bspw. als Key-Value-Paar empfangen wird.
+
+Wird bspw. das Topic `metering/total/kWh/waermepumpe` als Zählerstand konfiguriert und liefert den Wert `234.567` in Textform, dann muss der eigentliche Wert aus `waermepumpe=234.567` extrahiert werden. In diesem Fall wäre der reguläre Ausdruck also `(\d+.?\d*)`, um den Wert `234.567` im SAE weiter zu verarbeiten. Die RegEx kann bspw. via [RegEx101](https://regex101.com/) ermittelt werden. Dort die genannte RegEx sowie als Test-String `waermepumpe=234.567` einfügen und es wird direkt markiert, worauf die jeweilige RegEx matcht. Zudem wird genau erklärt, was die einzelnen Teile der RegEx effektiv machen.
```

Comment:
Der Inhalt des neuen Abschnitts sagt auch nichts anderes, als bereits in `ValueExtraction_DE.md`. Diese Inhalte habe ich auch absichtlich dorthin extrahiert, damit man es von überall refernezieren kann. Du hast jetzt hier nur für den MQTT-Zähler nochmal erklärt, wie man eine Regex verwendet.

Bestenfalls der erste Satz könnte hier stehenbleiben, wobei mich die Formulierung "`234.567` in Textform" eher irritiert als mir hilft. Meinst Du damit "`234.567` enthalten in einem Text"? Aber das steht auch schon so in `ValueExtraction_DE.md` und ist keine echt Zusatzinformation.

Reply:
Ok, dann braucht's da noch etwas Feintuning. :-)

Das Problem worüber ich gestolpert bin und was ich hier versucht habe zu klären ist, dass die anzugebende RegEx sich je nachdem was man unter "Format" gewählt hat, auf einen anderen Eingangswert bezieht! Sprich, wenn ich dort "JSON" wähle, dann wirkt die RegEx allein auf den Json-Content. Zumindest verstehe ich die Doku so, ich habe keinen Json-Content, an dem ich das gegenprüfen könnte. 

Wähle ich aber unter Format nichts aus, lasse es also leer, dann wirkt die RegEx auf den kompletten String, also "<key>=<value>"! Wäre dem nicht so, dann hätte es mit der RegEx `.*` auch funktionieren müssen, da das Value nur den Zahlenwert enthält. Erst durch den Hinweis in der Diskussion, dass ich mit der RegEx `(\d+.?\d*)` den Zahlenwert extrahieren muss, hat mich auf die Idee gebracht, dass hier der komplette String, also incl. "<key>=" durch die RegEx geschickt wird.

Dass das besser in `ValueExtraction_DE.md` gehört ist natürlich korrekt, mea culpa.

Reply:
Dein Verständnis ist richtig, d.h. `ValueExtraction_DE.md` sollte so angepasst werden, dass man diese Erkennthis gewinnen kann. Machst Du das noch in diesem PR, damit ich das mergen kann?

Reply:
Danke für die Bestätigung. Ich passe den PR selbstverständlich an.